atomicapp/nulecule/base.py

```diff
@@ -87,9 +88,20 @@ def unpack(cls, image, dest, config=None, namespace=GLOBAL_CONF,
             A Nulecule instance, or None in case of dry run.
         """
         logger.info('Unpacking image: %s to %s' % (image, dest))
-        docker_handler = DockerHandler(dryrun=dryrun)
-        docker_handler.pull(image)
-        docker_handler.extract(image, APP_ENT_PATH, dest, update)
+        if Utils.running_on_openshift():
+            _config = {
+                'providerapi': config['general']['providerapi'],
+                'accesstoken': config['general']['accesstoken'],
+                'namespace': config['general']['namespace']
+            }
+            op = OpenShiftProvider(_config, './', False)
```

Comment:
The reason being providers consume a **flat** config dictionary in contrast to the config data in **answers**. And our method ``get_context`` which does the conversion is an instance method written to create a flat **config** data for components. So, I used the above work around instead.